utility/src/main/java/org/oppia/util/gcsresource/GcsResouceAnnotations.kt

```diff
@@ -0,0 +1,16 @@
+package org.oppia.util.gcsresource
```

Comment:
Is this & the next file reversed? It seems like this contains the module despite being a file called annotations, and the module file contains an annotation. Please fix.